NEWS.md

```diff
@@ -1,5 +1,7 @@
 # rextendr (development version)
 
+* Claus Wilke is stepping down as a formal maintainer. Ilia Kosenkov is accepting the role of maintainer.
```

Comment:
What about simpler one like "Ilia Kosenkov is now the formal maintainer."?

e.g. https://github.com/r-lib/vroom/blob/main/NEWS.md#vroom-157

Reply:
Maybe "Ilia Kosenkov is now the official maintainer"?

It should definitely not say "stepping down as *a* formal maintainer", as there can be only one at a time, so it's "*the* formal maintainer".

Reply:
You are probably correct, I did not check how it is usually written in R packages. I'll replace it with what you think is the best option. If `{vroom}` example is OK, I can adopt that.

Reply:
I used the phrase you both kindly suggested.